members/MWAE480a6821.yaml

```diff
@@ -0,0 +1,41 @@
+bioguide: MWAE480a6821
+contact_form:
+  driver: phantom
+  steps:
+    - visit: "https://app.smartsheet.com/b/form/8e61f25fdfd1467bb016151a1529dd3e"
+    - find:
+        - selector: input[id='text_box_First Name']
```

Comment:
One note - I'm fairly certain IDs with whitespace are not valid html.  If this selector works in Phantom, I'm good with it and you can feel free to merge.  Just wanted to flag since it wouldn't be surprising to me if different apps/browsers didn't treat these as valid selectors even though it's in fact what's on the page.